modules/monitoring-osp.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * configuration/monitor-acs.adoc
+:_module-type: CONCEPT
+[id="monitoring-osp_{context}"]
+= Monitoring with {osp}
+
+Monitoring with {osp} is enabled by default. No configuration is required for this default behavior.
+
+[IMPORTANT]
+====
+If you have previously configured monitoring with the Prometheus Operator, consider removing your custom `ServiceMonitor` resources. {product-title-short} ships with a pre-configured `ServiceMonitor` for {osp} monitoring. Multiple `ServiceMonitors` may result in duplicated scraping.
+====
+
+Monitoring with {osp} is not supported by Scanner. If you want to monitor Scanner, you must first disable the default {osp} monitoring. Then, configure custom Prometheus monitoring.
+
+For more information on disabling {osp} monitoring, see "Disabling {osp} monitoring for Central services by using the {product-title-short} Operator" or "Disabling {osp} monitoring for Central services by using Helm". For more information on configuring Prometheus, see "Monitoring with custom Prometheus".
```

Comment:
How about having links to sections?
```suggestion
For more information on disabling {osp} monitoring, see xref:monitor-osp-disable-operator_{context}[Disabling {osp} monitoring for Central services by using the {product-title-short} Operator] or xref:monitor-osp-disable-helm_{context}[Disabling {osp} monitoring for Central services by using Helm]. For more information on configuring Prometheus, see "Monitoring with custom Prometheus".
```

Reply:
Unfortunately we can't have xrefs in modules - it works on docs.openshift.com but there is something on the Customer Portal (access.redhat.com) that breaks when docs are posted there that have xrefs in modules. That is why we have to do this clunky "Additional resources" thing. Noting that xrefs are OK in assemblies, just not modules.

Reply:
Here is the guidance if you are interested:

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references